lib/laa_crime_schemas/structs/income_details.rb

```diff
@@ -3,7 +3,9 @@
 module LaaCrimeSchemas
   module Structs
     class IncomeDetails < Base
-      attribute :benefits, Types::Array.of(Base) do
+      attribute :income_above_threshold, Types::String
```

Comment:
Shouldn't this be a boolean?
```
attribute :income_above_threshold, Types::Bool

Reply:
We're collecting it as a 'yes' or 'no' as an enum string. I'm happy to convert it to a bool on submission if that's the best way forward. Only set it as this because that's how we collect it. 
Same as this https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas/blob/a3064cefa57b95bc643ea110bd46cf62a7fc03dc/lib/laa_crime_schemas/structs/codefendant.rb#L6
I can at very least set it as : `attribute :income_above_threshold, Types::YesNoValue`

What do you think @timpeat ?